### ISSUE TITLE
Build failures with recent clang

### ISSUE THREAD

**Issue by PHHargrove** [clangupc/clang-upc#102]
With clang/clang++ from llvm-8.0 (and from Xcode 11) one cannot build clang-upc.

This was originally reported: [here](https://upc-bugs.lbl.gov/bugzilla/show_bug.cgi?id=3977#c1)

The work-around is the following "1-liner" which renames the parameter in the three places the problem occurs:

```
sed -i.bak \
        -e "6270,6280 s/CodeGenFunction &CGF/&_/" \
        -e "6316,6326 s/CodeGenFunction &CGF/&_/" \
        -e "6395,6405 s/CodeGenFunction &CGF/&_/" \
        tools/clang-upc/lib/CodeGen/CGOpenMPRuntime.cpp
```

This may not work with every version of awk, but works on macOS and Linux.

**Comment by PHHargrove** [comment]
Proposed fix in PR #103

**Comment by PHHargrove** [comment]
PR #103 was merged at commit e6ed41c on Sep 24, 2019
